### ISSUE TITLE
Unable to import component: No module named 'janus'

### ISSUE THREAD

**Issue by h4de5** [KaSroka/Toshiba-AC-control#47]
currently wondering if I am the only having this issue.

The following error occured since HA 2024.4:

```log
ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration 'toshiba_ac': Unable to import component: No module named 'janus'
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/homeassistant/loader.py", line 979, in async_get_component
    comp = await self.hass.async_add_import_executor_job(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/homeassistant/loader.py", line 1037, in _get_component
    ComponentProtocol, importlib.import_module(self.pkg_path)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/config/custom_components/toshiba_ac/__init__.py", line 4, in <module>
    from toshiba_ac.device_manager import ToshibaAcDeviceManager
  File "/config/.local/lib/python3.12/site-packages/toshiba_ac/device_manager.py", line 19, in <module>
    from toshiba_ac.device import ToshibaAcDevice
  File "/config/.local/lib/python3.12/site-packages/toshiba_ac/device/__init__.py", line 37, in <module>
    from toshiba_ac.utils.amqp_api import ToshibaAcAmqpApi
  File "/config/.local/lib/python3.12/site-packages/toshiba_ac/utils/amqp_api.py", line 21, in <module>
    from azure.iot.device.aio import IoTHubDeviceClient  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/.local/lib/python3.12/site-packages/azure/iot/device/aio/__init__.py", line 8, in <module>
    from azure.iot.device.iothub.aio import *  # noqa: F401, F403
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/.local/lib/python3.12/site-packages/azure/iot/device/iothub/aio/__init__.py", line 7, in <module>
    from .async_clients import IoTHubDeviceClient, IoTHubModuleClient
  File "/config/.local/lib/python3.12/site-packages/azure/iot/device/iothub/aio/async_clients.py", line 24, in <module>
    from .async_inbox import AsyncClientInbox
  File "/config/.local/lib/python3.12/site-packages/azure/iot/device/iothub/aio/async_inbox.py", line 8, in <module>
    import janus
ModuleNotFoundError: No module named 'janus'
```

I remember having problem with this module previously: #12 
latest version of azure-iot-device is 2.13 (https://pypi.org/project/azure-iot-device/)

**Comment by h4de5** [comment]
"fixed" it by adding "janus==1.0.0" inside the requirements list of HA manifest for toshiba_ac:

  "requirements": ["toshiba-ac==0.3.4","janus==1.0.0"]

**Comment by h4de5** [comment]
fixed in latest release https://github.com/h4de5/home-assistant-toshiba_ac/releases/tag/2024.6.0
